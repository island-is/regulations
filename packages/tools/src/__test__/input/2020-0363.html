<p style="text-align: center !important;">1. gr.</p>

<p style="text-align: justify;">Við reglugerðina bætast tvö ný bráðabirgðaákvæði svohljóðandi:</p>

<p style="text-align: center !important;">I.</p>

<p style="text-align: justify;">Þrátt fyrir 1. mgr. 5. gr. er Fiskistofu heimilt að taka til greina umsóknir um byggðakvóta sem bárust eftir umsóknarfrest í sveitarfélögum þar sem byggðakvóta hefur ekki þegar verið úthlutað.</p>

<p style="text-align: center !important;">II.</p>

<p style="text-align: justify;">Heimilt er sveitarstjórnum að óska eftir að tímabundið verði vikið frá skilyrðum 1. mgr. 6. gr. um að landa afla innan hlutaðeigandi byggðarlags ef vinnsla liggur niðri vegna COVID-19 að hluta eða öllu leyti.</p>

<p style="text-align: justify;">Sveitarstjórn skal senda tillögu til ráðuneytisins ásamt rökstuðningi um hvers vegna þörf er á tímabundinni undanþágu.</p>

<p style="text-align: justify;">&#160;</p>

<p style="text-align: center !important;">2. gr.</p>

<p style="text-align: justify;">Reglugerð þessi er sett samkvæmt ákvæðum laga nr. 79/1997, um veiðar í fiskveiðilandhelgi Íslands, laga nr. 116/2006, um stjórn fiskveiða og laga nr. 57/1996, um umgengni um nytjastofna sjávar og öðlast þegar gildi.</p>

<p style="text-align: justify;">&#160;</p>

<p style="text-align: center !important; font-style: italic !important;">Atvinnuvega- og nýsköpunarráðuneytinu, 21. apríl 2020.</p>

<p>&#160;</p>

<p style="text-align: center;"><strong>Kristján Þór Júlíusson</strong><br />
sjávarútvegs- og landbúnaðarráðherra.</p>

<div>
    <p style="text-align: right !important; font-style: italic !important;">Jóhann Guðmundsson.</p>
</div>

