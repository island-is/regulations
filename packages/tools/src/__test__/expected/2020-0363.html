<h3 class="article__title">
1.
gr.
</h3>

<p>
Við
reglugerðina
bætast
tvö
ný
bráðabirgðaákvæði
svohljóðandi:
</p>

<p align="center">
I.
</p>

<p>
Þrátt
fyrir
1.
mgr.
5.
gr.
er
Fiskistofu
heimilt
að
taka
til
greina
umsóknir
um
byggðakvóta
sem
bárust
eftir
umsóknarfrest
í
sveitarfélögum
þar
sem
byggðakvóta
hefur
ekki
þegar
verið
úthlutað.
</p>

<p align="center">
II.
</p>

<p>
Heimilt
er
sveitarstjórnum
að
óska
eftir
að
tímabundið
verði
vikið
frá
skilyrðum
1.
mgr.
6.
gr.
um
að
landa
afla
innan
hlutaðeigandi
byggðarlags
ef
vinnsla
liggur
niðri
vegna
COVID-19
að
hluta
eða
öllu
leyti.
</p>

<p>
Sveitarstjórn
skal
senda
tillögu
til
ráðuneytisins
ásamt
rökstuðningi
um
hvers
vegna
þörf
er
á
tímabundinni
undanþágu.
</p>

<h3 class="article__title">
2.
gr.
</h3>

<p>
Reglugerð
þessi
er
sett
samkvæmt
ákvæðum
laga
nr.
79/1997,
um
veiðar
í
fiskveiðilandhelgi
Íslands,
laga
nr.
116/2006,
um
stjórn
fiskveiða
og
laga
nr.
57/1996,
um
umgengni
um
nytjastofna
sjávar
og
öðlast
þegar
gildi.
</p>

<p class="Dags" align="center">
<em>Atvinnuvega-
og
nýsköpunarráðuneytinu,
21.
apríl
2020.</em>
</p>

<p align="center">
<strong>Kristján
Þór
Júlíusson</strong><br />
sjávarútvegs-
og
landbúnaðarráðherra.
</p>

<p align="right">
<em>Jóhann
Guðmundsson.</em>
</p>
